src/features/common/id_utils.ts

```diff
@@ -154,3 +154,8 @@ export async function getUserIdForEmail(email: string) {
   const response = await slack.client.users.lookupByEmail({ email: email });
   return response.user!.id!;
 }
+
+export async function getChannelIdForUser(userId: string) {
```

Comment:
Add a comment here that we get this when the user first opens the home screen.

(Also is there no other way in slack to get this?)

Reply:
Not really. It's really weird. Also discussed here: https://stackoverflow.com/questions/65648612/how-to-get-the-slack-dm-channel-id-of-the-slack-app